.golangci.yml

```diff
@@ -20,13 +20,14 @@ linters:
     - misspell
     - revive
     - staticcheck
-    - structcheck
     - typecheck
     - unused
-    - varcheck
```

Comment:
Removing these 3 linters to resolve the long standing warning from those be deprecated.